src/huntsman/pocs/observatory.py

```diff
@@ -711,7 +711,6 @@ def _take_flat_observation(self, exptimes, observation, fits_headers=None, dark=
         self.logger.debug(f'Slewing to flat-field coords: {observation.field}.')
         self.mount.set_target_coordinates(observation.field)
         self.mount.slew_to_target()
-        self.status()  # Seems to help with reading coords
```

Comment:
Does this need to be removed?

Reply:
It was raising errors because `self.status` returns a `dict` and is not callable. It shouldn't be necessary anyway. I've seen codes work without it...

Reply:
I have tried to make all `status` a property rather than a method so if we can just switch to that it would be better.

Reply:
But if that involves doing something to make it work then fine to delete here and handle in separate PR.